### PR TITLE
fixed req body

### PR DESCRIPTION
**Pull request by vinayak-trivedi**
changed create level request body due to the new changes in backend

### CODE REVIEW

File: src/admin-panel/index.js [hunk 1]
```diff
@@ -84,7 +83,7 @@ async function createLevel() {
     alert(err);
   } finally {
     removeLoadingState();
-    button.textContent = 'Create Level';
+    document.querySelector('.submit-button').textContent = 'Create Level';
```
Comment (ankushdharkar) [inline src/admin-panel/index.js:86]: Why was this change needed?

File: src/admin-panel/index.js [hunk 2]
```diff
@@ -118,7 +117,7 @@ async function createSkill() {
     alert(err);
   } finally {
     removeLoadingState();
-    button.textContent = 'Create Skill';
+    document.querySelector('.submit-button').textContent = 'Create Skill';
```
Comment (ankushdharkar) [inline src/admin-panel/index.js:120]: https://github.com/Real-Dev-Squad/website-dashboard/pull/263/files#r1038875892

**Review by ankushdharkar — CHANGES_REQUESTED** [review]

File: src/admin-panel/index.js (hunk 1 above)
Comment (vinayak-trivedi) [reply src/admin-panel/index.js:86]: because of my mistake in previous PR

Comment (ankushdharkar) [reply src/admin-panel/index.js:86]: On line 63, we already have a reference. `const button = document.querySelector('.submit-button');`, so why are we not using it here?

Comment (vinayak-trivedi) [reply src/admin-panel/index.js:86]: because it's scope is limited to the try block

**Review by ankushdharkar — APPROVED** [review]
